examples/java/pom.xml

```diff
@@ -95,7 +95,7 @@
       <dependencies>
         <dependency>
           <groupId>org.apache.beam</groupId>
-          <artifactId>beam-runners-flink_2.10</artifactId>
+          <artifactId>beam-runners-flink_${flink.scala.version}</artifactId>
```

Comment:
Just noting (brought up in person by @lukecwik) that this should go along with release instructions and whatnot. I don't know the status of that. It might be cleaner, depending on the capabilities of gradle and maven, to have no variable here but just explicit configs for supported versions.